tutorials/elixir-phoenix-on-google-app-engine.md

```diff
@@ -48,37 +56,238 @@ If you have not yet installed Elixir and Phoenix, do so:
     [Elixir install](https://elixir-lang.org/install.html) guides for your
     operating system.
 
-2. Install the hex and phx_new archives.
+2. Install the hex, rebar3, and phx_new archives.
 
         mix local.hex
-        mix archive.install https://github.com/phoenixframework/archives/raw/master/phx_new.ez
+        mix local.rebar
+        mix archive.install hex phx_new 1.4.0
 
 ## Create a new app and run it locally
 
-In this section, you will create a new Phoenix app and make sure it runs. If
-you already have an app to deploy, you may use it instead.
+In this section, you will create a new Phoenix app with a database, and make
+sure it runs locally in development. If you already have an app to deploy, you
+may use it instead.
 
-1.  Run the `phx.new` task to create a new Phoenix project called
-    "appengine_example". This tutorial will omit Ecto for now.
+### Create a new Phoenix app
 
-        mix phx.new appengine_example --no-ecto
+1.  Run the `phx.new` task to create a new Phoenix project called "hello".
```

Comment:
Elsewhere in the topic, you use code font for the name of the app: `hello`. Suggest doing the same  here.